Source/APIClient/Request.swift

```diff
@@ -19,6 +19,7 @@ public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
     static var authToken: String? { get }
+    static var authTokenHeader: String? { get }
```

Comment:
To @markdintrepid's point, I think that exposing it as a single variable would be more straightforward for the user. The existence of both variables could be confusing if it's not abundantly clear that `authorizationHeader` wants everything _but_ the token.